mlflow/spark.py

```diff
@@ -261,6 +261,8 @@ def log_model(
             code_paths,
             signature=signature,
             input_example=input_example,
+            pip_requirements=pip_requirements,
+            extra_pip_requirements=extra_pip_requirements,
```

Comment:
Found `pip_requirements` and `extra_pip_requirements` this when I was running code like this.

```python
mlflow.spark.log_model(..., pip_requirements=["pyspark"])
```

This code should skip dependency inference but it didn't.